dwitter/feed/views.py

```diff
@@ -81,7 +81,8 @@ def dweet_reply(request, dweet_id):
 
 
 @login_required
-def dweet_delete(request, dweet_id):
+def dweet_delete(request):
+    dweet_id = request.POST['dweet-id']
```

Comment:
Ideally you could use django forms to parse POST data in order to get free systematic validation of input.